### PR TITLE
[MRG] Upload Press Release Financing Round

### PR DESCRIPTION
**Pull request by Kirastina**

### CODE REVIEW

File: site/content/english/press-room.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ The Most Important Skribble Branding Assets are right here at hand:
 {{< /cta >}}
 
 {{< table top=12 width=narrow title="Media releases" >}}
+<tr>
+  <td>19.09.2019</td>
+  <td><strong>Second financing round closed (in German)</strong></td>
+  <td>
+    <a href="site/content/english/downloads/20200528_Skribble_Medienmitteilung_Abschluss_Finanzierungsrunde (5).pdf" target="_blank">Download</a>
```
Comment (Kirastina) [inline site/content/english/press-room.md:60]:
```suggestion
    <a href="downloads/20200528_Medienmitteilung_Abschluss_Finanzierungsrunde_Skribble.pdf" target="_blank">Download</a>
```

File: site/content/french/press-room.md [hunk 2]
```diff
@@ -54,6 +54,13 @@ Les atouts les plus importants de Skribble sont à portée de main :
 
 {{< table top=12 width=narrow title="Communiqués de presse" >}}
 
+<tr>
+  <td>28.05.2020</td>
+  <td><strong>Clôture du deuxième cycle de financement (en allemand)</strong></td>
+  <td>
+    <a href="site/content/english/downloads/20200528_Skribble_Medienmitteilung_Abschluss_Finanzierungsrunde (5).pdf" target="_blank">Download</a>
```
Comment (Kirastina) [inline site/content/french/press-room.md:61]:
```suggestion
    <a href="downloads/20200528_Medienmitteilung_Abschluss_Finanzierungsrunde_Skribble.pdf" target="_blank">Download</a>
```

File: site/content/german/press-room.md [hunk 3]
```diff
@@ -53,6 +53,13 @@ Die wichtigsten Skribble Branding Assets sind gleich hier griffbereit:
 {{< /cta >}}
 
 {{< table top=12 width=narrow title="Medienmitteilungen" >}}
+<tr>
+  <td>28.05.2020</td>
+  <td><strong>Zweite Finanzierungsrunde abgeschlossen</strong></td>
+  <td>
+    <a href="site/content/english/downloads/20200528_Skribble_Medienmitteilung_Abschluss_Finanzierungsrunde (5).pdf" target="_blank">Download</a>
```
Comment (Kirastina) [inline site/content/german/press-room.md:60]:
```suggestion
    <a href="downloads/20200528_Medienmitteilung_Abschluss_Finanzierungsrunde_Skribble.pdf" target="_blank">Download</a>
```

**Comment by michelfernandez** [comment]
Everything works. I'll checked the downloads.

**Pull request merged**